src/Compilers/CSharp/Test/Syntax/Parsing/UsingDirectiveParsingTests.cs

```diff
@@ -4077,4 +4037,1267 @@ public void TestMakeref()
         }
         EOF();
     }
+
+    [Fact]
+    public void TestUnsafeStatic1_CSharp11_NoUnsafeFlag()
+    {
+        var text = @"using unsafe static System.Console;";
+
+        CreateCompilation(text, parseOptions: TestOptions.Regular11).VerifyDiagnostics(
+            // (1,1): hidden CS8019: Unnecessary using directive.
+            // using unsafe static System.Console;
+            Diagnostic(ErrorCode.HDN_UnusedUsingDirective, "using unsafe static System.Console;").WithLocation(1, 1),
+            // (1,7): error CS8652: The feature 'using type alias' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+            // using unsafe static System.Console;
+            Diagnostic(ErrorCode.ERR_FeatureInPreview, "unsafe").WithArguments("using type alias").WithLocation(1, 7),
+            // (1,7): error CS0227: Unsafe code may only appear if compiling with /unsafe
+            // using unsafe static System.Console;
+            Diagnostic(ErrorCode.ERR_IllegalUnsafe, "unsafe").WithLocation(1, 7),
+            // (1,14): error CS9133: 'static' modifier must precede 'unsafe' modifier.
+            // using unsafe static System.Console;
+            Diagnostic(ErrorCode.ERR_BadStaticAfterUnsafe, "static").WithLocation(1, 14));
+
+        UsingTree(text,
```

Comment:
>UsingTree(text [](http://example.com/codeflow?start=8&length=14)

Is this using C# 11?

Reply:
at a parsing level no.  but i'll have it do that as well to demonstrate the same between semantics and syntax.